testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/adapter/servlet/CookieStoreRootContextTest.java

```diff
@@ -54,15 +53,6 @@
 @AppServerContainer(ContainerConstants.APP_SERVER_EAP6)
 public class CookieStoreRootContextTest extends DemoServletsAdapterTest {
 
-    // Javascript browser needed KEYCLOAK-4703
-    @Drone
-    @JavascriptBrowser
-    protected WebDriver jsDriver;
-
-    @Page
-    @JavascriptBrowser
-    protected OIDCLogin jsDriverTestRealmLoginPage;
-
```

Comment:
I'm not sure about this removal, are we sure the test works with htmlunit?

Reply:
Both Travis and Jenkins are green. I'm not sure how to perform additional checks...

Seems fine...

Reply:
I'm resolving this discussion. Please reopen it if you have any more concerns or ideas how to test it.